openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -735,6 +735,7 @@ This specification defines the following values for the `proof_type` property:
 
 * `jwt`: A JWT [@!RFC7519] is used as proof of possession. When `proof_type` is `jwt`, a `proof` object MUST include a `jwt` claim containing a JWT defined in (#jwt-proof-type).
 * `cwt`: A CWT [@!RFC8392] is used as proof of possession. When `proof_type` is `cwt`, a `proof` object MUST include a `cwt` claim containing a CWT defined in (#cwt-proof-type).
+* `ldp_vp_2.0`: A verifiable presentation signed using the Data Integrity Proof defined in [@VC_DATA_2.0], and where the proof of possession MUST be done in accordance with [@DI]. When `proof_type` is set to `ldp_vp_2.0`, the `proof` object MUST include a `ldp_vp` claim containing a [verifiable presentation](https://www.w3.org/TR/vc-data-model-2.0/#presentations-0) defined in (#ldp_vp-proof-type).
```

Comment:
you link to the V2 data model down here but above your example shows a v1 VC, you may want to update the above example to point to the v2 context and use v2 properties

Reply:
since there has been a rename from `linked data proofs` to `data integrity proofs` should this property be `di_vp_2.0`?

Reply:
I aligned to the spec, where Data Integrity Proofs are still named with Linked Data Proofs `ldp`. I am open to change that throughout the spec. But for now (the PR) i thought it would be better to stick to the used naming, as it should not touch the general naming imo.